OpenAPI/kiota/using.md

```diff
@@ -9,6 +9,18 @@ ms.date: 03/10/2023
 
 # Using the Kiota tool
 
+Kiota now supports the generation of AI plugins, including Type-B Plugin manifests, tailored for Microsoft Copilots. These AI plugins can be used to ground Large Language Models (LLMs) by integrating them with relevant, use-case-specific data, thereby enhancing the user experience.
```

Comment:
Official name is "Copilot Agent Plugins" CC @fabianwilliams

Reply:
@baywet @fabianwilliams, I'll make sure to give you a heads-up once the PR is ready for review.